docs/core/allocation.qmd

```diff
@@ -169,22 +153,10 @@ In the future new optimization objectives will be introduced, for demands of bas
 
 ### Basin demands
 
-- `quadratic_absolute`:
-$$
-    E_{\text{basin}} = \sum_{i \in B_S} \left( F_i^\text{basin in} - d_i^p(t)\right)^2
-$$
-- `quadratic_relative`:
-$$
-    E_{\text{basin}} = \sum_{i \in B_S} \left( 1 - \frac{F_i^\text{basin in}}{d_i^p(t)}\right)^2
-$$
-- `linear_absolute` (default):
+- `linear_absolute`:
 $$
     E_{\text{basin}} = \sum_{i \in B_S} \left| F_i^\text{basin in} - d_i^p(t)\right|
 $$
```

Comment:
I feel like the structure of this part of the documentation has to be a bit different now. Maybe the subsections `User Demands` and `Basin demands` can be removed, and you can add 2 short sentences to introduce each of the error terms (which now are for users and basins and in my PR I will add flow demands).

Reply:
I substituted the subsections with two sentences.